spring-integration-websocket/src/main/java/org/springframework/integration/websocket/ClientWebSocketContainer.java

```diff
@@ -85,6 +89,14 @@ public void setHeaders(HttpHeaders headers) {
 	}
 
 	/**
+	 * Set the connection timeout in seconds; default: 10.
+	 * @param connectionTimeout the timeout in seconds.
```

Comment:
`@since 4.2`, but minor, of course. 
Although it may be exposed to XML attribute as well